Makefile

```diff
@@ -94,11 +95,31 @@ e2e-tests-es: prepare-e2e-tests es
 	@echo Running Elasticsearch end-to-end tests...
 	@STORAGE_NAMESPACE=$(STORAGE_NAMESPACE) go test -tags=elasticsearch ./test/e2e/... -kubeconfig $(KUBERNETES_CONFIG) -namespacedMan ../../deploy/test/namespace-manifests.yaml -globalMan ../../deploy/crds/jaegertracing_v1_jaeger_crd.yaml -root .
 
+.PHONY: e2e-tests-self-provisioned-es
+e2e-tests-self-provisioned-es: prepare-e2e-tests deploy-es-operator
+	@echo Running Self provisioned Elasticsearch end-to-end tests...
+	@go test -tags=self_provisioned_elasticsearch ./test/e2e/... -kubeconfig $(KUBERNETES_CONFIG) -namespacedMan ../../deploy/test/namespace-manifests.yaml -globalMan ../../deploy/crds/jaegertracing_v1_jaeger_crd.yaml -root .
+
 .PHONY: run
 run: crd
 	@rm -rf /tmp/_cert*
 	@bash -c 'trap "exit 0" INT; OPERATOR_NAME=${OPERATOR_NAME} KUBERNETES_CONFIG=${KUBERNETES_CONFIG} WATCH_NAMESPACE=${WATCH_NAMESPACE} go run -ldflags ${LD_FLAGS} main.go start'
 
+.PHONY: set-max-map-count
+set-max-map-count:
+	@minishift ssh -- 'sudo sysctl -w vm.max_map_count=262144' > /dev/null 2>&1 || true
+
+.PHONY: deploy-es-operator
+deploy-es-operator: set-max-map-count
+	@kubectl create namespace ${ES_OPERATOR_NAMESPACE} 2>&1 | grep -v "already exists" || true
+	@kubectl apply -f https://raw.githubusercontent.com/openshift/elasticsearch-operator/master/manifests/01-service-account.yaml -n ${ES_OPERATOR_NAMESPACE}
```

Comment:
from @jpkrohling  

> Could you use a specific revision or a tag? We don't want our tests to work today and fail tomorrow because of a change in upstream.

Unfortunately there is no release/tag in https://github.com/openshift/elasticsearch-operator. Using SHA does not solve the problem either because it will pick up the latest image which might require changes in deployment - e.g. recently roles were modified and added p8s dependency.

Reply:
We're getting closer to a release - @jcantrill will know more